changelog.d/20221007_180414_benjaminpoldrack_fix_6599.md

```diff
@@ -0,0 +1,58 @@
+<!-- Uncomment the section that is right (remove the HTML comment wrapper).-->
+<!--
+### Breaking Changes
+
+- Describe change, possibly reference closed/related issue/PR.
+  Fixes https://github.com/datalad/datalad-next/issues/XXX via
+  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+-->
+<!--
+### Enhancements and New Features
+
+- Describe change, possibly reference closed/related issue/PR.
+  Fixes https://github.com/datalad/datalad-next/issues/XXX via
+  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+-->
+
+### Bug Fixes
+
+- Avoid writing to stdout/stderr from within datalad sshrun. This could lead to
+  broken pipe errors when cloning via SSH and was superfluous to begin with.
+  Fixes https://github.com/datalad/datalad/issues/6599 via
+  https://github.com/datalad/datalad/pull/7072 (by @bpoldrack)
+
+<!--
+### Dependencies
+
+- Describe change, possibly reference closed/related issue/PR.
+  Fixes https://github.com/datalad/datalad-next/issues/XXX via
+  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+-->
+<!--
+### Documentation
+
+- Describe change, possibly reference closed/related issue/PR.
+  Fixes https://github.com/datalad/datalad-next/issues/XXX via
+  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+-->
+<!--
+### Internal
+
+- Describe change, possibly reference closed/related issue/PR.
+  Fixes https://github.com/datalad/datalad-next/issues/XXX via
+  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+-->
+<!--
+### Performance
+
+- Describe change, possibly reference closed/related issue/PR.
+  Fixes https://github.com/datalad/datalad-next/issues/XXX via
+  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+-->
+<!--
+### Tests
+
+- Describe change, possibly reference closed/related issue/PR.
+  Fixes https://github.com/datalad/datalad-next/issues/XXX via
+  https://github.com/datalad/datalad-next/pull/XXX (by @GITHUBHANDLE)
+-->
```

Comment:
remove all other stubs (even if scriv to remove them, why bother carrying in the diffs etc)?